packages/dds/tree/src/simple-tree/core/treeNodeKernel.ts

```diff
@@ -134,6 +135,7 @@ export class TreeNodeKernel implements Listenable<KernelEvents> {
 	/**
 	 * Create a TreeNodeKernel which can be looked up with {@link getKernel}.
 	 *
+	 * @param initialContext - context from when this node was originally crated.
```

Comment:
Nit: I would suggest putting these docs on the inline param, rather than here as an `@param` block. Intellisense should respect that for both the property and the constructor param.

Reply:
IntelliSense respect this for both just fine already.

If I move it, then I only see it for the property, and not in the IntelliSense for the constructor.

Maybe I'm missing something, but it seems like you have this backwards, and making the change causes it to only work in one case when it works in both as is.

Reply:
Hmm. That wasn't my experience in the past. But the fact that we also have standard constructor parameters with `@param` docs could also have an effect.